Frontend/ui-customer/src/Screens/OnBoardingFlow/EnterMobileNumberScreen/ComponentConfig.purs

```diff
@@ -69,9 +71,10 @@ whatsAppOTPButtonConfig state = let
       , enableLoader = (JB.getBtnLoader "PrimaryButtonMobileNumber")
       , stroke = ("1," <> Color.borderColorLight)
       , isSuffixImage = true
+      , visibility = if (getMerchant FunctionCall) == YATRISATHI then VISIBLE else GONE
```

Comment:
@S-PRAVINKUMAR use a config variable which can be fetched from v1-configuration and used , instead of adding merchant check.